docker-compose.jaeger.yml

```diff
@@ -0,0 +1,88 @@
+version: '3.7'
+
+services:
+  sia:
+    environment:
+      - SIA_MODULES=gctwr
+      - JAEGER_DISABLED=${JAEGER_DISABLED:-true}  # Enable/Disable tracing
+      - JAEGER_SERVICE_NAME=${PORTAL_NAME:-Skyd}  # change to e.g. eu-ger-1
+      # Configuration
+      # See https://github.com/jaegertracing/jaeger-client-go#environment-variables
+      # for all options.
+      - JAEGER_SAMPLER_TYPE=const
+      - JAEGER_SAMPLER_PARAM=1
+      - JAEGER_AGENT_HOST=jaeger-agent
+      - JAEGER_AGENT_PORT=6831
+      - JAEGER_REPORTER_LOG_SPANS=false
+    depends_on:
+      - jaeger-agent
+
+  jaeger-agent:
+    image: jaegertracing/jaeger-agent
+    command: [ "--reporter.grpc.host-port=jaeger-collector:14250", "--reporter.grpc.retry.max=1000" ]
+    container_name: jaeger-agent
+    restart: on-failure
+    ports:
+      - "5775:5775/udp"
+      - "6831:6831/udp"
+      - "6832:6832/udp"
+      - "5778:5778"
```

Comment:
Shouldn't those be `localhost:5775:5775` etc. to avoid docker adding them to iptables?

Maybe even better use 
```
expose:
  - someport
```
To not export the port at all and only make it visible from within the shared network of the containers?
https://github.com/compose-spec/compose-spec/blob/master/spec.md#expose

Reply:
Same goes for all the other exposed ports in this file